CHANGELOG.md

```diff
@@ -7,6 +7,198 @@ documentation before upgrading to a new release.
+| icingacli-businessprocess        | --ack-is-ok                | icingacli\_businessprocess\_ackisok      | #9103 |
+| icingacli-businessprocess        | --blame                    | icingacli\_businessprocess\_blame        | #9103 |
+| icingacli-businessprocess        | --colors                   | icingacli\_businessprocess\_colors       | #9103 |
+| icingacli-businessprocess        | --downtime-is-ok           | icingacli\_businessprocess\_downtimeisok | #9103 |
+| icingacli-businessprocess        | --root-cause               | icingacli\_businessprocess\_rootcause    | #9103 |
+| mem                              | -a                         | mem\_available                           | #9385 |
+| mongodb                          | --disable\_retry\_writes   | mongodb\_disableretrywrites              | #9539 |
+| mongodb                          | --ssl-ca-cert-file         | mongodb\_ssl\_ca\_cert\_file             | #9610 |
+| mysql                            | --extra-opts               | mysql\_extra\_opts                       | #9197 |
+| nrpe                             | -3                         | nrpe\_version\_3                         | #9296 |
+| nrpe                             | -D                         | nrpe\_no\_logging                        | #9016 |
+| nrpe                             | -P                         | nrpe\_payload\_size                      | #9032 |
+| pgsql                            | --extra-opts               | pgsql\_extra\_opts                       | #9197 |
+| postgres                         | $PGCONTROLDATA (env. var.) | postgres\_pgcontroldata                  | #8929 |
+| postgres                         | --datadir                  | postgres\_datadir                        | #8924 |
+| postgres                         | --language                 | postgres\_language                       | #8924 |
+| postgres                         | --perflimit                | postgres\_perflimit                      | #8924 |
+| ssl\_cert                        | --ignore-host-cn           | ssl\_cert\_ignore\_host\_cn              | #9512 |
+| ssl\_cert                        | --ignore-ocsp-errors       | ssl\_cert\_ignore\_ocsp\_errors          | #9512 |
+| ssl\_cert                        | --ignore-ocsp-timeout      | ssl\_cert\_ignore\_ocsp\_timeout         | #9512 |
+| ssl\_cert                        | --ignore-tls-renegotiation | ssl\_cert\_ignore\_tls\_renegotiation    | #9042 |
+| ssl\_cert                        | --proxy                    | ssl\_cert\_proxy                         | #8927 |
+| tcp                              | --sni                      | tcp\_sni                                 | #9347 |
+| vmware-esx-dc-runtime-tools      | --no\_vm\_tools\_ok        | vmware\_novmtools                        | #9611 |
+| vmware-esx-soap-vm-runtime-tools | --no\_vm\_tools\_ok        | vmware\_novmtools                        | #9611 |
```

Comment:
Can we use ``` `foo_bar` ``` instead of `foo\_bar` instead? IMHO this would make it more pleasant to read in the plain text form and also is more grep-friendly.

Reply:
I could do this, but this would enlarge the table further. see also doc/ITL/disk

Reply:
> see also doc/ITL/disk

Either I don't understand what you're referring to or what I should see there.